html/Mogo/js/popup/popup.js

```diff
@@ -0,0 +1,39 @@
+import {components} from './components.js';
+export function addPopup() {
+  components.then(({body, gallery, popup, popupOverlay, imgToShow}) => {
+    gallery.addEventListener('click', (event) => {
+      let target = event.target;
+      if (target.classList.contains('gallery__details-box')) {
+        const srcPath = event.target.previousElementSibling.getAttribute('src');
+
+        // eslint-disable-next-line no-param-reassign
+        popup.style.top = window.scrollY + 'px';
+
+        popup.classList.add('gallery__show--active');
+
+        imgToShow.setAttribute('src', srcPath);
+
+        body.classList.add('popup-active');
+      }
+    });
+
+    gallery.addEventListener('mouseout', (event) => {
```

Comment:
Can you show this block using only scss and :hover, without js evets?

Reply:
Btw lets add 'cursor: pointer' when user hovers the image. Because it's clickable